### PR TITLE
Update domain to .box

### PR DESCRIPTION
**Pull request by ChaituVR**

### CODE REVIEW

File: services/sx.js.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ const ethSigClient = new clients.EthereumSig(clientConfig);
 import { clients } from '@snapshot-labs/sx';
 import { Provider, constants } from 'starknet';
 
-const ethUrl = 'https://rpcs.snapshotx.xyz/1';
+const ethUrl = 'https://rpcs.snapshot.box/1';
```
Comment (wa0x6e) [inline services/sx.js.md:58]: This url does not exist

Comment (ChaituVR) [reply services/sx.js.md:58]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

File: services/sx.js.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ import { Provider, constants } from 'starknet';
 
 const web3 = window.starknet.provider;
 
-const ethUrl = 'https://rpcs.snapshotx.xyz/1';
+const ethUrl = 'https://rpcs.snapshot.box/1';
```
Comment (ChaituVR) [inline services/sx.js.md:160]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

File: services/sx.js.md [hunk 3]
```diff
@@ -364,7 +364,7 @@ import { Provider, constants } from 'starknet';
...
```
Comment (ChaituVR) [inline services/sx.js.md:367]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

File: services/sx.js.md [hunk 4]
```diff
@@ -199,7 +199,7 @@ import { Provider, constants } from 'starknet';
...
```
Comment (ChaituVR) [inline services/sx.js.md:202]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

File: services/sx.js.md [hunk 5]
```diff
@@ -479,7 +479,7 @@ import { Provider, constants } from 'starknet';
...
```
Comment (ChaituVR) [inline services/sx.js.md:482]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

File: services/sx.js.md [hunk 6]
```diff
@@ -518,7 +518,7 @@ import { Provider, constants } from 'starknet';
...
```
Comment (ChaituVR) [inline services/sx.js.md:521]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

File: services/sx.js.md [hunk 7]
```diff
@@ -320,7 +320,7 @@ import { Provider, constants } from 'starknet';
...
```
Comment (ChaituVR) [inline services/sx.js.md:323]:
```suggestion
const ethUrl = 'https://rpc.snapshot.box/1';
```

**Comment by wa0x6e** [comment]
https://rpc.snapshot.box/ is not good either

**Comment by ChaituVR** [comment]
> https://rpc.snapshot.box/ is not good either

opps. fixed it

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**